src/banman.h

```diff
@@ -34,6 +34,16 @@ class CSubNet;
 // between nodes running old code and nodes running
 // new code.
 
+
+enum class BanLevel {
+    // 0 - Not banned
+    NOT_BANNED                  =    0,
+    // 1 - Automatic misbehavior ban
+    AUTOMATIC_MISBEHAVIOUR_BAN  =    1,
+    // 2 - Any other ban
+    ANY_OTHER_BAN               =    2
```

Comment:
nit: comments seem redundant; if kept, could use the in-line doxygen format: `//!<`

Reply:
Removed, as you told they were redundant